### ISSUE TITLE
Create a release

### ISSUE THREAD

**Issue by berliner** [dekoder-org/donation-encourager#6]
This is a very nice package and I'm in the process of creating a [Drupal module](https://www.drupal.org/project/donation_encourager) for easier integration and setup.
It would help immensely if you could create a release, as that would make it a lot easier for the less tech-savvy Drupal folks to download and install it.
For testing I had already forked your repo and played around with creating a release, see here for example: https://github.com/berliner/donation-encourager/releases
But for obvious reasons I would very much prefer to just point users to your repo.

Thanks and let me know if there is anything I can help with.

**Comment by berliner** [comment]
Any kind of feedback would be nice.

**Comment by berliner** [comment]
Really, just any feedback ...

**Comment by berliner** [comment]
Ping

**Comment by daniel-marcus** [comment]
Hi @berliner and sorry for the late answer! Is [that](https://github.com/dekoder-org/donation-encourager/releases/tag/v1.2.0) something you can operate with already?

**Comment by berliner** [comment]
@daniel-marcus Yes, that's definitely a good first step. I need to refresh my memory on this and compare to the additional changes I made in my fork. I'll come back to you on this. Thanks for replying!
